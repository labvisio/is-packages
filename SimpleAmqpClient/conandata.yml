sources:
  "2.5.0":
    url: "https://github.com/alanxz/SimpleAmqpClient/archive/v2.5.0.tar.gz"
    sha256: "ba7d6bfb2ac0fc31d5c98bf103f180e3ed3dd9a5902de533fd94417e15c577a6"