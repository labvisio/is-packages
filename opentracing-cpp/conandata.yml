sources:
  "1.4.0":
    url: https://github.com/opentracing/opentracing-cpp/archive/v1.4.0.tar.gz
    sha256: 2eb0a4a7dc62bc8cbf12872080197b41d53b4c04966c860774a6b11fd59fad55