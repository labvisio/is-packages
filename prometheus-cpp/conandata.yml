sources:
  "0.4.1":
    url: "https://github.com/jupp0r/prometheus-cpp/archive/v0.4.1.tar.gz"
    sha256: "ce95e1387c69711666699594caf67fa98edc7d06025ced50a629fc92f1c83565"
patches:
  "0.4.1":
    - patch_file: "patches/0001-include-limits.patch"