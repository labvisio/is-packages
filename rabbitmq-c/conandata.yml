sources:
  "0.9.0":
    url: "https://github.com/alanxz/rabbitmq-c/archive/v0.9.0.tar.gz"
    sha256: "316c0d156452b488124806911a62e0c2aa8a546d38fc8324719cd29aaa493024"