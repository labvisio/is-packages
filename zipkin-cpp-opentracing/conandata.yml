sources:
  "0.3.1":
    url: "https://github.com/rnburn/zipkin-cpp-opentracing/archive/v0.3.1.tar.gz"
    sha256: "f16a6f1eed494ca3c2607d7ad671cb134bd7eb320c5969c8281c10922a146589"