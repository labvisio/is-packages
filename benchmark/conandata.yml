sources:
  "1.4.1":
    url: https://github.com/google/benchmark/archive/refs/tags/v1.4.1.tar.gz
    sha256: f8e525db3c42efc9c7f3bc5176a8fa893a9a9920bbd08cef30fb56a51854d60d