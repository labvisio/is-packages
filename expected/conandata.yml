sources:
  "0.3.0":
    url: "https://github.com/TartanLlama/expected/archive/v0.3.tar.gz"
    sha256: "18103ac213b3a7e5740bee754c9a5f96a4cee0ead1f2f7670775efc467b56ac3"